index.html

```diff
@@ -3462,20 +3463,23 @@ <h2>
             <dd>
 A <a href="metadata-structure">metadata structure</a> consisting of values
 relating to the results of the <a>DID URL Dereferencing</a> process. This
-structure is REQUIRED and MUST NOT be empty.
+structure is REQUIRED and in the case of an error in the dereferencing process,
+this MUST NOT be empty.
 Properties defined by this specification are in 
 <a href="#did-url-dereferencing-metadata-properties"></a>.
 If the dereferencing is not successful, this structure MUST contain an
 <code>error</code> property describing the error.
             </dd>
             <dt>
-content
+content-stream
             </dt>
             <dd>
 If the <code>dereferencing</code> function was called and successful, this MUST
-contain a resource corresponding to the <a>DID URL</a>.
-The content MAY be a <a>DID document</a> in one of the conformant <a href="#representations">representations</a>.
-obtained through the resolution process.
+contain a resource corresponding to the <a>DID URL</a>. 
+Especially in the case that the <a>DID URL</a> is a <a>DID</a>, the
+<code>content-stream</code> MAY be a <a>DID document</a> in one of the
+conformant <a href="#representations">representations</a> obtained through
+the resolution process.
```

Comment:
```suggestion
contain a resource corresponding to the <a>DID URL</a>.
The <code>content-stream</code> MAY be a <a>DID document</a> in one of the
conformant <a href="#representations">representations</a> obtained through
the resolution process.
```

As discussed before, even though I agree with your text, I think it's better if we avoid concrete statements about what exactly the DID URL dereferences to in which cases.